# Benthos Release Command Message
int64 timestamp
int8 id
string command
